ingest-asset-reconciler/src/test/scala/uk/gov/nationalarchives/ingestassetreconciler/LambdaTest.scala

```diff
@@ -233,7 +233,7 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach with TableDrivenPro
     argumentVerifier.verifyInvocationsAndArgumentsPassed(1, 0, 0, 0, 1, 0)
   }
 
-  "handler" should "return an error if more than one entity with the same asset name as SourceIdis" in {
+  "handler" should "return an error if more than one entity has the same asset name as SourceIdis" in {
```

Comment:
1. There's a spelling mistake in "SourceId"
2. could you change the ending to "as its SourceID"?